build.py

```diff
@@ -198,6 +198,81 @@ def get_names():
     def get_dict():
         return dict(Project._dict)
 
+    @staticmethod

```

Comment:
I try all day to fix the ^M and I think, at the end, that is only a display issue that happen when the line has @ in it, I don't know if only at the beginning (or after the initial spaces).

In a comment doesn't happen nothing, in the middle the last two character of the line are red (if numeric) and something like this :(

I tried to merge locally and the final result seems ok (there are only the CRLF at the end of the @staticmethod).

A a test if you look at the file build.py with github also in the other @staticmethod, at the end, the ^M is show.